src/styles/helpers/normalize.scss

```diff
@@ -0,0 +1,7 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
```

Comment:
[STYLES] - Remember to use fallback fonts - alternative font-family in case the main one doesn't work [like this](https://www.w3schools.com/cssref/pr_font_font-family.asp)
![image](https://github.com/mate-academy/layout_catalog/assets/63243136/1045775b-b487-49e5-a625-10e5e162785c)